src/main/kotlin/inc/evil/d17map/dtos/ClassroomResponse.kt

```diff
@@ -10,5 +11,6 @@ data class ClassroomResponse(
     val capacity: Int,
     val equipmentIds: Set<UUID>,
     val id: UUID,
-    val floor: Floor
+    val floor: Floor,
```

Comment:
Tutaj pytanko czy w response chcemy zwracać cały Building / Floor? Czy np. wystarczyłoby zwrócić `floor.name`¿ Być może to znowu pytanie do frontu 😅

Reply:
Przeoczenie - już poprawiam na nazwy